servers/v3-proxy-api/src/graph/get/toGraphQL.ts

```diff
@@ -195,28 +195,37 @@ export function setSavedItemsVariables(
 ): UserSavedItemsByOffsetArgs {
   const filter = SavedItemsFilterFactory(requestParams);
   const sort = SavedItemsSortFactory(requestParams);
-  return {
+  const variables = {
     pagination: {
       limit: requestParams.count,
       offset: requestParams.offset,
     },
     sort,
-    filter,
   };
+  // Attach filter only if not empty
+  if (Object.keys(filter).length) {
+    variables['filter'] = filter;
+  }
+  return variables;
 }
 
 export function setSearchVariables(
   requestParams: V3GetParams,
 ): UserSearchSavedItemsByOffsetArgs {
   const sort = SearchSortFactory(requestParams);
   const filter = SearchFilterFactory(requestParams);
-  return {
+
+  const variables = {
     term: requestParams.search,
     pagination: {
       limit: requestParams.count,
       offset: requestParams.offset,
     },
     sort,
-    filter,
   };
+  // Attach filter only if not empty
+  if (Object.keys(filter).length) {
```

Comment:
i kinda wish graphql would be ok with empty objects so clients didnt have to do this kind of thing.

Reply:
I thought it would since all the params are optional, but it's rejecting it :(